modules/drivers/bigquery-cloud-sdk/test/metabase/driver/bigquery_cloud_sdk/query_processor_test.clj

```diff
@@ -228,10 +228,12 @@
             " `v3_test_data.venues`.`name` AS `name` "
             "FROM `v3_test_data.venues` "
             "LIMIT 1")
-           (tt/with-temp* [Database [db    {:engine :bigquery-cloud-sdk
+           (tt/with-temp* [Database [db    {:engine  :bigquery-cloud-sdk
                                             :details (assoc (:details (mt/db))
                                                             :include-user-id-and-hash false)}]
-                           Table    [table {:name "venues" :db_id (u/the-id db)}]
+                           Table    [table {:name   "venues"
+                                            :db_id  (u/the-id db)
+                                            :schema (get-in db [:details :dataset-filters-patterns])}]
```

Comment:
this looks weird. the singular schema is `dataset-filters-patterns`? that sounds plural. Also, aren't there exclusion patterns? Would the schema end up with the pattern to _exclude_?

Reply:
Yeah, so, there is a single String field that can contain multiple patterns (comma separated).  See [here](https://www.notion.so/metabase/Sync-multiple-schemas-with-controls-621d075ca25c47d1b6170fb5688da2b3) for examples and mockups.  That single value (which, again, is comma separated) can either represent inclusion xor exclusion, depending on another selection box.

Reply:
isn't that a bad choice for the `:schema` of a table then?